{"name": "Запрос с несуществующим номером возвращает ошибку", "status": "passed", "steps": [{"name": "Генерация случайного id", "status": "passed", "start": 1717008331194, "stop": 1717008331194}], "start": 1717008331194, "stop": 1717008333257, "uuid": "4eec2457-f020-446f-bd93-d9e07547137f", "historyId": "3a35d5e78d8b1287eccfea83d0fad245", "testCaseId": "3a35d5e78d8b1287eccfea83d0fad245", "fullName": "tests.test_order_get_by_track.TestGetOrderByTrack#test_get_order_bad_track_negative", "labels": [{"name": "feature", "value": "Получение заказа по его номеру"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_order_get_by_track"}, {"name": "subSuite", "value": "TestGetOrderByTrack"}, {"name": "host", "value": "DESKTOP-JN5AD1H"}, {"name": "thread", "value": "4908-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_order_get_by_track"}]}